aws-opentelemetry-distro/src/amazon/opentelemetry/distro/patches/_bedrock_patches.py

```diff
@@ -238,5 +241,164 @@ def extract_attributes(self, attributes: _AttributeMapT):
+            if 'tokenCount' in result:
+                span.set_attribute(GEN_AI_USAGE_OUTPUT_TOKENS, result['tokenCount'])
+            if 'completionReason' in result:
+                span.set_attribute(GEN_AI_RESPONSE_FINISH_REASONS, [result['completionReason']])
+    
+    def _handle_anthropic_claude_response(self, span: Span, response_body: Dict[str, Any]):
+        if 'usage' in response_body:
+            usage = response_body['usage']
+            if 'input_tokens' in usage:
+                span.set_attribute(GEN_AI_USAGE_INPUT_TOKENS, usage['input_tokens'])
+            if 'output_tokens' in usage:
+                span.set_attribute(GEN_AI_USAGE_OUTPUT_TOKENS, usage['output_tokens'])
+        if 'stop_reason' in response_body:
+            span.set_attribute(GEN_AI_RESPONSE_FINISH_REASONS, [response_body['stop_reason']])
+
+    def _handle_cohere_command_response(self, span: Span, response_body: Dict[str, Any]):
+        # Input tokens: Approximate from the user's message in chat history
+        if 'chat_history' in response_body:
+            user_messages = [msg['message'] for msg in response_body['chat_history'] if msg['role'] == 'USER']
+            input_text = ' '.join(user_messages)
+            span.set_attribute(GEN_AI_USAGE_INPUT_TOKENS, math.ceil(len(input_text) / 6))
+        # Output tokens: Approximate from the response text
+        if 'text' in response_body:
+            span.set_attribute(GEN_AI_USAGE_OUTPUT_TOKENS, math.ceil(len(response_body['text']) / 6))
+        if 'finish_reason' in response_body:
+            span.set_attribute(GEN_AI_RESPONSE_FINISH_REASONS, [response_body['finish_reason']])
+
+    def _handle_ai21_jamba_response(self, span: Span, response_body: Dict[str, Any]):
+        print("This is the response body :", response_body)
+        if 'usage' in response_body:
+            usage = response_body['usage']
+            if 'prompt_tokens' in usage:
+                span.set_attribute(GEN_AI_USAGE_INPUT_TOKENS, usage['prompt_tokens'])
+            if 'completion_tokens' in usage:
+                span.set_attribute(GEN_AI_USAGE_OUTPUT_TOKENS, usage['completion_tokens'])
+        if 'choices' in response_body:
+            choices = response_body['choices'][0]
+            if 'finish_reason' in choices:
+                span.set_attribute(GEN_AI_RESPONSE_FINISH_REASONS, [choices['finish_reason']])
+
+    def _handle_meta_llama_response(self, span: Span, response_body: Dict[str, Any]):
+        #print("This is the response body :", response_body)
+        if 'prompt_token_count' in response_body:
+            span.set_attribute(GEN_AI_USAGE_INPUT_TOKENS, response_body['prompt_token_count'])
+        if 'generation_token_count' in response_body:
+            span.set_attribute(GEN_AI_USAGE_OUTPUT_TOKENS, response_body['generation_token_count'])
+        if 'stop_reason' in response_body:
+            span.set_attribute(GEN_AI_RESPONSE_FINISH_REASONS, response_body['stop_reason'])
```

Comment:
super small but the GEN_AI_RESPONSE_FINISH_REASONS for meta llama should be wrapped inside an array

Reply:
Hey Steve,
Thanks for catching that, I have updated the code